app/components/provider_interface/edit_provider_user_permissions_component.html.erb

```diff
@@ -0,0 +1,38 @@
+<legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
```

Comment:
Future PR: I think this should go into the `<form>` (specifically `govuk_check_boxes_fieldset`).

Reply:
Ok, will keep in mind.